app/views/api/v1/profiles/update.json.jbuilder

```diff
@@ -4,7 +4,7 @@ json.uid @user.uid
 json.name @user.name
 json.nickname @user.nickname
 json.email @user.email
-json.account_id @user.account_id
+json.account_id @user.account.id
```

Comment:
account_id?

Reply:
@pranavrajs this is intentional.  @user.account_users.where(condition).account gives the account. we have a method written in the user `account` to fetch account directly from this relation.